### DISCUSSION TITLE
Multi level column headers

### DISCUSSION THREAD

**Discussion by abi-kannan** [Megabit/Blazorise#2521] [Q&A]
I have a grid, I want to add multiple level of column header ( group columns ) like this 

![multilevel_header](https://user-images.githubusercontent.com/43766343/123326940-d5008300-d507-11eb-8d53-6cd2c4c60adb.PNG)

I tried adding DataGridColumns under a DataGridColumn, but it is not working. 
```
<DataGridColumns>
    <DataGridColumn TItem="OrderDetails" Caption="OrderId" Field="OrderId" />
	<DataGridColumn TItem="OrderDetails" Caption="Order Details" >
		<DataGridColumn TItem="OrderDetails" Caption="Order Date" Field="OrderDate" />
		<DataGridColumn TItem="OrderDetails" Caption="Freight" Field="Freight" />
	</DataGridColumn>
		<DataGridColumn TItem="OrderDetails" Caption="Ship Details" >
		<DataGridColumn TItem="OrderDetails" Caption="Shipped Date" Field="ShippedDate" />
		<DataGridColumn TItem="OrderDetails" Caption="Shipped Country" Field="Country" />
	</DataGridColumn>
</DataGridColumns>
```

Is there a way to achieve this ?

**Comment by stsrki** [comment]
This is not possible with DataGrid. Your best option is to make it yourself by using Table and its subparts.
